app.rb

```diff
@@ -6,6 +6,23 @@
 
 module MojFile
   class Uploader < Sinatra::Base
+    configure do
+      set :raise_errors, true
+      set :show_exceptions, false
+    end
+
+    get '/healthcheck' do
+      {
+        dependencies: {
+          external: {
+            s3: {
+              S3::REGION.tr('-','_') => S3.status
+            }
+          }
+        }
+      }.to_json
+    end
+
```

Comment:
@digitalronin I was thinking they should probably just be amalgamated into a single call, but wanted to talk to Leo about it, first.